### PR TITLE
update testnet ranges for replay-verify

### PR DESCRIPTION
**Pull request by aluon**
## Description

Update testnet ranges to exclude tapos transactions

## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## How Has This Been Tested?

Will run replay-verify manually https://github.com/aptos-labs/aptos-core/actions/runs/9714483252

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by perryjrandall** [comment]
Test plan just links to all jobs, not a specific job?

File: testsuite/replay_verify.py [hunk 1]
```diff
@@ -16,25 +16,26 @@
...
+    (1_115_000_000, 1_128_000_000),
+    (1_128_000_000, 1_141_000_000),
+    (1_141_000_000, 1_154_000_000),
+    (1_154_000_000, 1_167_000_000),
     (1_167_000_000, 1_183_000_000),
     (1_183_000_000, 1_199_000_000),
     (1_199_000_000, 1_215_000_000),
     (1_215_000_000, 1_230_000_000),
     (1_230_000_000, 1_246_000_000),
-    (1_246_000_000, 1_262_000_000),
```
Comment (perryjrandall) [inline testsuite/replay_verify.py:37]: we probably dont wanna remove this shard, just skip tapos transactinos (starts roughly around 1259000000)

Comment (aluon) [reply testsuite/replay_verify.py:37]: I ran another workflow to test setting the end of the range https://github.com/aptos-labs/aptos-core/actions/runs/9714540293.

For this PR I just deleted the last range and added another one at the beginning. I also reduced the size for some of the longer running ranges

**Review by areshand — APPROVED** [review]

**Review by sherry-x — APPROVED** [review]

File: testsuite/replay_verify.py [hunk 2]
```diff
@@ -196,9 +196,8 @@ def main(runner_no=None, runner_cnt=None, start_version=None, end_version=None):
     ), "runner_cnt must match the number of runners in the mapping"
     runner_start = runner_mapping[runner_no][0]
     runner_end = runner_mapping[runner_no][1]
-    latest_version = query_backup_latest_version(BACKUP_CONFIG_TEMPLATE_PATH)
+    query_backup_latest_version(BACKUP_CONFIG_TEMPLATE_PATH)
```
Comment (perryjrandall) [inline testsuite/replay_verify.py:199]: lets change the name to make this more obvious

**Review by perryjrandall — APPROVED** [review]

**Pull request merged**